README.md

```diff
@@ -48,6 +49,7 @@ forecasting - which is notorious for the size and complexity of its workflows.
 ![PyPI](https://img.shields.io/pypi/pyversions/cylc-flow.svg?color=green)
 [![PyPI](https://img.shields.io/pypi/v/cylc-flow.svg?color=yellow)](https://pypi.org/project/cylc-flow/)
 [![Anaconda-Server Badge](https://anaconda.org/conda-forge/cylc-flow/badges/version.svg)](https://anaconda.org/conda-forge/cylc-flow)
+[![Documentation](https://img.shields.io/website?label=documentation&up_message=live&url=https%3A%2F%2Fcylc.github.io%2Fcylc-doc%2Flatest%2Fhtml%2Findex.html)](https://cylc.github.io/cylc-doc/latest/html/index.html)
```

Comment:
Latest is still Cylc7:

```suggestion
[![Documentation](https://img.shields.io/website?label=documentation&up_message=live&url=https%3A%2F%2Fcylc.github.io%2Fcylc-doc%2Flatest%2Fhtml%2Findex.html)](https://cylc.github.io/cylc-doc/nightly/html/index.html)
```

Reply:
Latest is 8.0a2? https://cylc.github.io/cylc-doc/latest/html/index.html

Once we release the b0 docs, this link will point to that.

Most users will be on the beta as opposed to master, so I'd have thought this will be better pointing to latest?

Reply:
Oh, ok, thought "latest" would be 7.9.3 rather than an alpha release.